br/cmd/br/backup.go

```diff
@@ -31,6 +31,15 @@ func runBackupCommand(command *cobra.Command, cmdName string) error {
 		ctx, store = trace.TracerStartSpan(ctx)
 		defer trace.TracerFinishSpan(ctx, store)
 	}
+
+	if cfg.FullBackupType == task.FullBackupTypeEBS {
+		if err := task.RunBackupEBS(ctx, tidbGlue, &cfg); err != nil {
+			log.Error("failed to backup", zap.Error(err))
```

Comment:
a little concern here, zap.Error(err) may print a call stack and above return errors.Trace(err) may also print in caller. can we remove it?